src/pharmpy/tools/estmethod/algorithms.py

```diff
@@ -28,7 +35,14 @@ def exhaustive(methods, solvers, parameter_uncertainty_methods):
 def exhaustive_with_update(methods, solvers, parameter_uncertainty_methods):
     wb = WorkflowBuilder()
 
-    task_base_model = Task('create_base_model', _create_base_model)
+    add_eval_after_est = len(methods) > 1
+
+    task_base_model = Task(
+        'create_base_model',
+        _create_base_model,
+        parameter_uncertainty_methods[0],
```

Comment:
Good point about questioning the use of `parameter_uncertainty_methods[0]` for the base model. I chose this because that index is always guaranteed to be available, and it's efficient to use one of the `parameter_uncertainty_methods` that we're already testing. This way, we avoid running extra models that aren't part of the experiments. For example, if we're testing `IMP` and `FOCE` methods along with `SANDWICH` and `CPG` for parameter uncertainty, choosing `None` would add an extra run with `FOCE` and no parameter uncertainty—this isn't efficient.

However, there's a case I hadn't considered. If `FOCE` isn't in the search space and `parameter_uncertainty_methods` isn't `None`, then it's unnecessary to use one of the parameter uncertainty methods for the base model. The base model is for estimation, not for evaluating parameter uncertainty. So, I've added some logic to handle this scenario:

```python
parameter_uncertainty_method_base = parameter_uncertainty_methods[0] if 'FOCE' in methods else None
```

This way, if FOCE isn't part of the search space, the base model won't use any of the parameter uncertainty methods, which makes more sense. Thanks for bringing this to my attention!